CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ versioning. Rather than a static releases, this repository contains of a number
 of regression tests that are each semi-independent.  This CHANGELOG file should be used
 to document pull requests to this repository.
 
+## 2024-11-04 ([#107](https://github.com/nasa/harmony-regression-tests/pull/112))
```

Comment:
Super nit: This should have `[#112]`, I think.

Reply:
it should [1d79e97](https://github.com/nasa/harmony-regression-tests/pull/112/commits/1d79e970f9dfcdf86e13be5b801da64524ddd180)